open_spiel/python/mfg/algorithms/softmax_policy.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Computes a softmax policy from a value."""
+import numpy as np
+
+from open_spiel.python import policy as policy_std
```

Comment:
We don't seem to be using a policy variable, so it may be simpler to just use import policy instead of aliasing to policy_std.

Reply:
You're right. I let this because I copied the file greedy_policy.py. Do you want me to change it also in greedy_policy.py?

Reply:
Thanks! We can perhaps change it later in another PR. It is not very important and better to keep the scope of the PR limited.